script/tool/lib/src/native_test_command.dart

```diff
@@ -622,3 +674,15 @@ class _PlatformResult {
   /// Ignored unless [state] is `failed`.
   final String? error;
 }
+
+/// The state of a .java test file.
+class _JavaTestInfo {
+  const _JavaTestInfo(
+      {required this.usesFlutterTestRunner,
+      required this.hasDartIntegrationTestAnnotation});
+
+  /// Whether the test class uses the FlutterTestRunner runner.
+  final bool usesFlutterTestRunner;
+  //// Whether the test class has the @DartIntegrationTest annotation.
```

Comment:
A little too commented for my liking.

Reply:
Heh, I was about to defend the comment as being marginally useful even though it's not much more information than the name has, until I realized that you meant it had an extra `/`. Good eye; fixed!